src/app.js

```diff
@@ -77,7 +77,7 @@ export default function (app) {
 
           } else {
             for (i = 0; i < hooks.onUpdate.length; i++) {
-              hooks.onUpdate[i](model, result, data)
+              hooks.onUpdate[i](model, result, data, actions)
```

Comment:
Ideally yes 👍 for the sakes of not introducing breaking changes that could be avoided, perhaps no? Or in a separate PR?